config/jobs/kubernetes/sig-scalability/sig-scalability-presets.yaml

```diff
@@ -164,7 +164,7 @@ presets:
     value: "--kube-reserved=cpu=1050m"
   - name: KUBEPROXY_TEST_ARGS
     # TODO(#74011): Remove metrics-bind-address if the default is set.
-    value: "--profiling --metrics-bind-address=0.0.0.0"
+    value: "--profiling --metrics-bind-address=0.0.0.0 --feature-gates=MinimizeIPTablesRestore=true"
```

Comment:
Does the flag exist in all minors?

The preset-e2e-scalability-common is being used by all the minors. We can set it here and override back in the non-master tests using this presubmit.

Reply:
I think that is better to add it to the periodic jobs, it's a scale optimization and it will be interesting to see the tread

Reply:
ping @mborsz @wojtek-t  any advice on how to proceed?

it can not run on previous versions, also from this comment seems @wojtek-t have a better idea on what kind of workloads it should be running https://github.com/kubernetes/enhancements/pull/3454/files#r979617245

Reply:
Sorry, I wasn't clear. I agree on adding this to periodic job.
My concern here is that we if simply put this here, it will populate to all minors and potentially break pre 1.26 tests.

I think suggested approach is to:
* Add it here
* In config/jobs/kubernetes/sig-release/release-branch-jobs/{1.22...1.25}.yaml in all jobs using preset-e2e-scalability-common add - --env=KUBEPROXY_TEST_ARGS="--profiling --metrics-bind-address=0.0.0.0" (so value before the override)

Alternative approach would be to add this as --env=KUBEPROXY_TEST_ARGS only to current master tests (1.26 already), but there is quite a lot of them.

Reply:
ok, got it now, I didn't get it before XD